src/MgvRoot.sol

```diff
@@ -35,55 +36,56 @@ contract MgvRoot is HasMgvEvents {
     MgvStructs.OfferPacked offer;
     MgvStructs.OfferDetailPacked detail;
   }
-  /* `Pair` contains the information specific to an oriented `outbound_tkn,inbound_tkn` pair:
+  /* `OfferList` contains the information specific to an oriented `outbound_tkn,inbound_tkn`, `tickScale` offerList:
 
-    * `local` is the Mangrove configuration specific to the `outbound,inbound` pair. It contains e.g. the minimum offer `density`. It contains packed information, see [`structs.js`](#structs.js) for more.
+    * `local` is the Mangrove configuration specific to the `outbound,inbound,tickScale` offerList. It contains e.g. the minimum offer `density`. It contains packed information, see [`structs.js`](#structs.js) for more.
     * `offerData` maps from offer ids to offer data.
   */
 
   /* Note that offers are structured into a tree with linked lists at its leves.
      The root is level2, has 256 level1 node children, each has 256 level0 node children, each has 256 leaves, each has 4 ticks (it holds the first and last offer of each tick's linked list).
      level2, level1 and level0 nodes are bitfield, a bit is set iff there is a tick set below them.
   */
-  struct Pair {
+  // FIXME rename OfferList to OLD (means OfferListData) or such
```

Comment:
I must admit that the renaming from Pair to OfferList made it quite hard to find the important bits to review so I am not sure I saw it all :)

Reply:
Well the FIXME is an old comment I meant to remove :p `OLKey` is really everywhere / pervasive, so a short name would be better. Also `key` I think is less error-inducing maybe? `Id` works as well. Just `Key` maybe?

Reply:
Alright. I think we're entering bike-shedding territory, especially re. length of names 😅
Re. just using `Key`: It's a generic name that doesn't convey what it relates to. So if the name should be ultra-short I think the current name or an acronym is be better as it's less easy to misunderstand.